packages/test-utils/src/testUtils/TestApiProvider.test.tsx

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { createApiRef, useApiHolder } from '@backstage/core-plugin-api';
+import { TestApiProvider } from './TestApiProvider';
+import { render, screen } from '@testing-library/react';
+import { TestApiRegistry } from '.';
+
+const xApiRef = createApiRef<{ a: string; b: number }>({
+  id: 'x',
+});
+const yApiRef = createApiRef<string>({
+  id: 'y',
+});
+
+function Verifier() {
+  const holder = useApiHolder();
+  const x = holder.get(xApiRef);
+  const y = holder.get(yApiRef);
+
+  return (
+    <div>
+      {x ? (
+        <span>
+          x={x.a},{x.b}
+        </span>
+      ) : (
+        <span>no x</span>
+      )}
+      {y ? <span>y={y}</span> : <span>no y</span>}
```

Comment:
missed opportunity to call this `u` and `no u`

Reply:
but y

Reply:
tho